### ISSUE TITLE
sls help should ignore most validations and print help instantly

### ISSUE THREAD

**Issue by eahefnawy** [serverless/serverless#1721]
If there's an issue in your service config, running `sls help` shouldn't throw an error. Like the following example.

```
$ sls help

  Serverless Error ---------------------------------------

     Region "us-east-1" doesn't exist in stage "dev"

  Get Support --------------------------------------------
     Docs:          v1.docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
```

cc/ @flomotlik @pmuens

**Comment by pmuens** [comment]
Just looked into it today.

The problem with this one is that we rely / have to wait for the `Service` class to parse the YAML so that we can display what plugins are used and the corresponding help on how to display help for the plugins. If there's an error in this parsing we won't see / reach the help as the error is thrown previously.

Furthermore we want to track whether people heave entered the `help` command which is also done after parsing the YAML file (because we're also tracking the loaded plugins etc.)

Here's the code for more clarification on this one:
https://github.com/serverless/serverless/blob/master/lib/Serverless.js#L75-L78

As we're dropping the `serverless.env.yml` file as a strict dependency which caused this error I would vote to close this issue for now as it's not that important right now IMHO.

**Comment by eahefnawy** [comment]
Hmmm agreed. This change will be directly affected by #1801 ... any effort we put here might be obsolete after #1801 . I'd say we put it on hold for now since it's not super critical.

**Comment by eahefnawy** [comment]
also what you mentioned makes sense, if we want to display help for custom plugins that are defined in `serverless.yml`, we obviously need to parse `serverelss.yml` first, so if there's a parsing issue, there's not much we can do here. I think the best we could do is just to limit the errors as much as possible and display the help as soon as possible while bypassing any irrelevant validations

**Comment by jordanmack** [comment]
The same issue also exists for `--version`.

**Comment by pmuens** [comment]
@jordanmack yep. The `--help` and `--version` commands use the same method behind the scenes.

**Comment by dougmoscrop** [comment]
I believe this is fixed? https://github.com/serverless/serverless/issues/2041

**Comment by pmuens** [comment]
Done with #2133 /cc @flomotlik
